addons/binding/org.openhab.binding.omnilink/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="omnilink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Unit channels -->
+
+	<channel-type id="unit_status">
+		<item-type>Number</item-type>
+		<label>Status</label>
+		<description>Represents the numeric status of an unit object</description>
+		<state pattern="%d" readOnly="true" />
+	</channel-type>
+	<channel-type id="unit_level">
+		<item-type>Dimmer</item-type>
+		<label>Light Level</label>
+		<description>Increase/decrease the light level</description>
+		<category>DimmableLight</category>
+		<state min="0" max="100" pattern="%d %%" />
+	</channel-type>
+	<channel-type id="unit_switch">
+		<item-type>Switch</item-type>
+		<label>Load Switch</label>
+		<description>Toggle Load</description>
+		<category>Switch</category>
+		<state pattern="%s" />
+	</channel-type>
+	<channel-type id="scene_switch">
+		<item-type>Switch</item-type>
+		<label>On/Off</label>
+		<description>Send On/Off to a Scene</description>
+		<category>Switch</category>
+		<state pattern="%s" />
+	</channel-type>
+	<channel-type id="scene_on">
+		<item-type>Switch</item-type>
+		<label>On Scene</label>
+		<description>Turn on On Scene</description>
+		<category>Switch</category>
+		<state pattern="%s" />
+	</channel-type>
+	<channel-type id="scene_off">
+		<item-type>Switch</item-type>
+		<label>Off Scene</label>
+		<description>Turn on Off Scene</description>
+		<category>Switch</category>
+		<state pattern="%s" />
+	</channel-type>
+	<channel-type id="scene_toggle">
+		<item-type>Switch</item-type>
+		<label>Scene Toggle</label>
+		<description>Scene Toggle</description>
+		<category>Toggle</category>
+		<state pattern="%s" />
+	</channel-type>
+	<channel-type id="room_state">
+		<item-type>Number</item-type>
+		<label>State</label>
+		<description>Status of room (0 off, 1 on, 2 scene a, 3 scene b, etc)</description>
+		<state pattern="%s">
+			<options>
+				<option value="0">Off</option>
+				<option value="1">On</option>
+				<option value="3">Scene A</option>
+				<option value="4">Scene B</option>
+				<option value="5">Scene C</option>
+				<option value="6">Scene D</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="flagvalue">
```

Comment:
I need to change this to "flag_value" to keep with the format of the rest of the channels